templates/rails/app/views/layouts/application.html.erb.tt

```diff
@@ -47,11 +47,5 @@
   <%%= yield %>
 
   <%%= javascript_include_tag 'application' %>
-  <%%= javascript_tag do %>
-    $(function(){
-      $(document).foundation();
-    });
-  <%% end %>
-  <%%= yield :extra_footer %>
```

Comment:
It likely requires explaining in the documentation but the intent is to allow for students to inject javascript at the end of the `body` tag. Javascript should always be the last thing loaded on the page. In view logic you can do the following:

```ruby
<%= content_for :extra_footer do %>
<script type="text/javascript">
  var widget = new Something.Widget('foo');
</script>
<% end %>
```